src/utils/getDecimalSeparator.ts

```diff
@@ -0,0 +1,10 @@
+import ccI18n from "../i18n";
+
+const getDecimalSeparator = (): string => {
```

Comment:
`getSeperator("decimal")` does the same as this function

Reply:
means replace all occurances of this function with the above mentioned and remove it here

Reply:
@TQjona  - Removed this file